src/view/Dialog.less

```diff
@@ -0,0 +1,57 @@
+@import "~nav-frontend-core/less/core";
+
+.dialog {
+  horiz-align: right;
```

Comment:
Tror det mangler noen bokstaver her.

Reply:
fjernet (syntaksen er riktig ifølge Intellij men hadde ingen effekt her)

Reply:
Testet den akkurat her, og intelliJ klaget ikke her heller. 
Men jeg finner ingen referanser til det attributtet i css-specen; https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference 

Testet i tillegg om less-kompilatoren, eller postCSS kanskje brukte det som en shorthand for noe annet. Men ingen av de to ser ut til å reagere på den heller. 

Så jeg mistenker at den kanskje kan tilhøre en tidligere spesifikasjon, eller at en spesifik browser kanskje har støtte for den (MS/IE var jo kjente for å base finne på masse rart tidligere).